hw/ip/rv_core_ibex/doc/boot-rom-patching.md

```diff
@@ -140,7 +140,7 @@ The second ROM partition with patching capability is added to the architecture d
 
 - **The patch table** includes:
 
-  - Contents for 32 [*patch match register pairs*](#patch-match--redirect-logic), each register 1 DWORD in size.
+  - Contents for 32 [*patch match descriptor*](#patch-match--redirect-logic), each descriptor is 2 DWORDs in size.
```

Comment:
Nit: Missing "s" after `descriptor` (to make it a plural)

Reply:
Ah, good eyes :-) Thanks, fixed now.